app/livechat/server/hooks/sendToCRM.js

```diff
@@ -54,8 +55,7 @@ function sendToCRM(type, room, includeMessages = true) {
 			if (message.t === msgNavType) {
 				msg.navigation = message.navigation;
 			}
-
-			postData.messages.push(msg);
+			postData.messages.push(message.file ? FileUpload.addJWTToFileUrlIfNecessary(message) : msg);
```

Comment:
As I commented before, IMO it would be better having a specific helper method to handle this process, and inside that method, I would call the addJWTToFileUrlIfNecessary method.
In that case, we could rename the addJWTToFileUrlIfNecessary to addJWTToFileUrl.